draft-bbs-signatures.md

```diff
@@ -770,10 +762,15 @@ The cipher-suites in Section 4 are based upon the BLS12-381 pairing-friendly ell
 
 * subgroup_check MAY use either the naive check described in Section 1.3 or the optimized check given by [Bowe19].
 
-## Test Vectors
+### Test Vectors
 
 //TODO
 
+# IANA Considerations
+
+This document does not make any requests of IANA.
```

Comment:
we need a spec that does update the JOSE / COSE registry... eventually. 
I would prefer if we could do that here.

Reply:
See #48